src/components/PackageList/PackageList.tsx

```diff
@@ -13,16 +13,16 @@ interface Props {
 }
 
 export const PackageList: React.FC<Props> = props => {
```

Comment:
Please destruct the props here, then you don't need to to like this: 
![image](https://user-images.githubusercontent.com/29228205/67668964-3d811080-f971-11e9-8598-90029878c917.png)

Reply:
we are not destructuring the props for this component.

Reply:
Thank you!! :-)

Reply:
I would still destruct the props like this: `const PackageList: React.FC<Props> =({ packages }) => `

Reply:
ahhh I see what you mean! done.